### DISCUSSION TITLE
Rebranding

### DISCUSSION THREAD

**Discussion by cl8dep** [cl8dep/depingaasere-wiki#8] [Ideas]
Propuse un nuevo rebranding al proyecto basado en los colores de la bandera. 

Tomando como Primary Color el color azul de la bandera: #002A8F

**Comment by DarielF** [comment]
Y los íconos ?

**Comment by DarielF** [comment]
**Lista de cosas por hacer para el rebranding:**

- [ ] Escoger colores (principal y secundario)
- [ ]  Icono de la página
- [ ] Imágenes de la página de inicio
- [ ] Guías de diseño (identidad): comprendería cómo añadir cualquier elemento visual respetando la armonía visual e identidad del sitio.
